packages/common/package.json

```diff
@@ -30,9 +29,6 @@
       "codegen": [
         "./src/codegen/index.ts"
       ],
-      "deprecated": [
-        "./src/deprecated/index.ts"
-      ],
```

Comment:
since there's `@deprecated` typedoc operator that does a better job of signaling deprecated stuff, I am preferring that now over marking this as deprecated in the export path